ecommerce/api.py

```diff
@@ -200,6 +203,18 @@ def fulfill_completed_order(
     )
 
 
+def add_fulfilled_order_to_paid_courserun(order):
+    """
+    When order is fulfilled or reviewed, record it to paid courserun table if not exist
+    """
+    order = Order.objects.get(id=order.id)
+    if order.STATE.FULFILLED or order.STATE.REVIEW:
+        for run in order.purchased_runs:
+            PaidCourseRun.objects.get_or_create(
+                order=order, course_run=run, user=order.purchaser
+            )
```

Comment:
This could probably be a method on `Order` so it can just use `self`. I'd also say we already have a property `is_review` so we should use that and also add `is_fulfilled`.

Probably worth adding ones for the other states too and updating any usages, but that's probably best as a separate improvement issue.